nengo/conftest.py

```diff
@@ -210,12 +231,17 @@ def pytest_runtest_setup(item):
                 ('logger', 'logs', "logs not requested")]:
             if fixture_name in item.fixturenames:
                 if item.config.getvalue(option):
-                    skip = False
+                    break
                 else:
                     skipreasons.append(message)
-        if skip:
+        else:
             pytest.skip(" and ".join(skipreasons))
 
+    if 'Simulator' in item.fixturenames:
+        for test, reason in _Simulator.unsupported:
```

Comment:
Is there a better way to do this search? I'm guessing it's not a big performance issue, but we are searching through all the unsupported things for each test. I guess because we want to use regex we have to do it this way? Nothing better is coming to mind for me, but it's something to think about, and maybe fix down the road.